Users guide generation warning

**Is your feature request related to a problem? Please describe.**
Generated from user's guide action from cFS
```
/home/runner/work/cFS/cFS/osal/src/os/inc/osapi-bsp.h:95: warning: end of file while inside a group
/home/runner/work/cFS/cFS/osal/src/os/inc/osapi-idmap.h:274: warning: end of file while inside a group
```

**Describe the solution you'd like**
Resolve warnings

**Describe alternatives you've considered**
None

**Additional context**
None

**Requester Info**
Jacob Hageman - NASA/GSFC

--- a/src/os/inc/osapi-bsp.h
+++ b/src/os/inc/osapi-bsp.h
@@ -91,4 +91,6 @@ char *const *OS_BSP_GetArgV(void);
  ------------------------------------------------------------------*/
 void OS_BSP_SetExitCode(int32 code);
 
+/**@}*/
+
 #endif

--- a/src/os/inc/osapi-idmap.h
+++ b/src/os/inc/osapi-idmap.h
@@ -51,15 +51,6 @@
 #define OS_OBJECT_TYPE_USER        0x10 /**< @brief Object user type */
 /**@}*/
 
-/** @defgroup OSAPICore OSAL Core Operation APIs
- *
- * These are for OSAL core operations for startup/initialization, running, and shutdown.
- * Typically only used in bsps, unit tests, psps, etc.
- *
- * Not intended for user application use
- * @{
- */
-
 /** @defgroup OSAPIObjUtil OSAL Object ID Utility APIs
  * @{
  */

